src/pages/billing/SpendReport/CostCard.ts

```diff
@@ -8,12 +8,12 @@ interface CostCardProps {
   amount: string
   isProjectCostReady: boolean
   showAsterisk: boolean
-  key: string
+  type: string
```

Comment:
It turned out that using "key" didn't work because it's specially treated in React components.

Reply:
I've definitely chased that rabbit before